README.md

```diff
@@ -43,6 +43,10 @@ The current version is **4.2.7**. The `master` branch on GitHub is version
 
 ## Installation
 
+**Note:** In order to use the features described in the current
+[documentation](http://bourbon.io/docs) you will need to specify version **4.2.7**
+as the gem version in each of the installation options below.
```

Comment:
I think this message can go away. v5.0 is in beta, so you'd have to purposely pass a flag to install a beta. Any installation using the means below with install the current stable version, which is 4.2.7.